docs/extend/modular-resources/create/_index.md

```diff
@@ -84,6 +84,13 @@ The following example module registers a modular resource implementing Viam's bu
 
 The Go code for the custom model (<file>mybase.go</file>) and module entry point file (<file>main.go</file>) is adapted from the full demo modules available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources).
 
+{{% alert title="Naming module models" color="tip" %}}
+
+Name your model with all lowercase letters for optimal performance with Viam's SDKs.
```

Comment:
Oh! I did not know about this bug!

I have [this PR](https://github.com/viamrobotics/docs/pull/1705) for model naming guidance, which adds some context to this section, and points to a new section on the Concepts page.

I think this guidance (lowercase) would better live there alongside the other related guidance I've added there, like triplet form, namespace matching, matching model `family` to API `subtype`, etc.

Do you think providing guidance in the above-referenced PR is the place for this? If you still feel strongly that we also need notice here, given the changes in my PR, I'm happy to review this on its own. Otherwise, I suggest that I take this ticket's ask and fold it into my existing naming PR. Whatcha think? Happy either way!